docs/guides/CustomStartupGuide.md

```diff
@@ -361,7 +389,4 @@ Query parameter:
 curl http://localhost:59882/api/v2/device/name/<device name>/StreamingStatus | jq -r '"StreamingStatus: " + (.event.readings[].objectValue.IsStreaming|tostring)'
 ```
 
-If the StreamingStatus is false, the camera is not configured to stream video. Please try the Start Video Streaming section again.
-
-## License
-[Apache-2.0](LICENSE)
+If the StreamingStatus is false, the camera is not configured to stream video. Please try the Start Video Streaming section again [here](#start-video-streaming).
```

Comment:
I think this troubleshooting section is where the V4L2 compilation errors should go.